packetbeat/packetbeat.template-es2x.json

```diff
@@ -37,59 +40,79 @@
               "properties": {}
             },
             "auto-delete": {
+              "doc_values": true,
+              "index": true,
               "type": "boolean"
             },
             "class-id": {
+              "doc_values": true,
+              "index": true,
               "type": "long"
             },
             "consumer-count": {
+              "doc_values": true,
+              "index": true,
               "type": "long"
             },
             "consumer-tag": {
+              "doc_values": true,
               "ignore_above": 1024,
               "index": "not_analyzed",
               "type": "string"
             },
             "content-encoding": {
+              "doc_values": true,
               "ignore_above": 1024,
               "index": "not_analyzed",
               "type": "string"
             },
             "content-type": {
+              "doc_values": true,
```

Comment:
As long as the defaults stay the same, there is no need to add it. Lets remove this too for the moment to keep it the same as it was previously and introduce defaults in case we need it. Defaults "should have been there" in the python script but were ignored ...